personal_assistant/pa_contacts/urls.py

```diff
@@ -1,11 +1,12 @@
 from django.urls import path
 
-from .views import main, create, delete
+from .views import main, create, delete, edit
 
 app_name = 'pa_contacts'
 
 urlpatterns = [
     path('', main, name='main'),
     path('add/', create, name='create'),
     path('<int:contact_id>/delete', delete, name='delete'),
+    path('<int:contact_id>/edit', edit, name='edit'),
```

Comment:
Так як немає шляху для контактів лише з одним їх ідентифікатором тож пропоную закрити цю прогалину спростивши доданий:
```suggestion
    path('<int:contact_id>/', edit, name='edit'),
```